src/SMAIAXBackend.API/Program.cs

```diff
@@ -46,20 +51,42 @@
 
     using var scope = app.Services.CreateScope();
     var services = scope.ServiceProvider;
+
     var applicationDbContext = services.GetRequiredService<ApplicationDbContext>();
+    var tenantRepository = services.GetRequiredService<ITenantRepository>();
+    var tenantDbContextFactory = services.GetRequiredService<ITenantDbContextFactory>();
+
     await applicationDbContext.Database.EnsureDeletedAsync();
     await applicationDbContext.Database.EnsureCreatedAsync();
     await applicationDbContext.SeedTestData();
+
+    // Create a database for tenant_1 with test data for development
+    var dbConfig = app.Configuration.GetSection("DatabaseConfiguration").Get<DatabaseConfiguration>();
+    var tenantDbContext =
+        tenantDbContextFactory.CreateDbContext("tenant_1_db", dbConfig!.SuperUsername, dbConfig.SuperUserPassword);
+    await tenantDbContext.Database.EnsureDeletedAsync();
+    await using (var deleteUserCommand = applicationDbContext.Database.GetDbConnection().CreateCommand())
+    {
+        deleteUserCommand.CommandText = "DROP ROLE IF EXISTS johndoe;";
+        await applicationDbContext.Database.OpenConnectionAsync();
+        await deleteUserCommand.ExecuteNonQueryAsync();
+        await applicationDbContext.Database.CloseConnectionAsync();
+    }
+    await tenantRepository.CreateDatabaseForTenantAsync("tenant_1_db", "johndoe", "P@ssw0rd");
```

Comment:
shouldn't we read username and password from a file as opposed to having it hardcoded? Even if it's just for the test db?